blocks.md

```diff
@@ -251,22 +269,16 @@ Key | Values | Required | Default
 `critical` | Minimum usage, where state is set to critical. | No | `90`
 `interval` | Update interval, in seconds. | No | `1`
 `format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{utilization}"`
-`per_core` | Display CPU frequencies and utilization per core. | No | `false`
```

Comment:
There's no way to keep this around is there?

Reply:
I don't think there is.

Reply:
Let's document these breaking changes in NEWS.md. I added this one to the file to start with.